contrib/uthash/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "uthash"
+pkgver = "2.3.0"
+pkgrel = 0
+hostmakedepends = ["gmake", "perl"]
+pkgdesc = "C macros for hash tables and more"
+maintainer = "ttyyls <contact@behri.org>"
+license = "BSD-2-Clause"
+url = "https://troydhanson.github.io/uthash"
+source = (
+    f"https://github.com/troydhanson/uthash/archive/refs/tags/v{pkgver}.tar.gz"
+)
+sha256 = "e10382ab75518bad8319eb922ad04f907cb20cccb451a3aa980c9d005e661acc"
+options = ["empty"]
```

Comment:
a headers-only package doesn't need to have a -devel subpackage

```suggestion
```

Reply:
sure

Reply:
you still need to remove this line :)